x-pack/packages/kbn-cloud-security-posture/common/utils/helpers.ts

```diff
@@ -62,3 +62,43 @@ export const buildEntityFlyoutPreviewQuery = (field: string, queryValue?: string
     },
   };
 };
+
+export const buildEntityAlertsQuery = (field: string, queryValue?: string, size?: number) => {
+  return {
+    size: size || 0,
+    _source: false,
+    fields: [
```

Comment:
we don't need alerts itself for the preview, we only need to aggregation of the alerts on severity. Even though you need the alerts for the data grid in the expanded flyout, I think it should be a separate query. For preview we need to get only that data we need for this preview. This also goes together with he ask for using exactly the same logic as preview on document flyout